### ISSUE TITLE
HeadNoteWaitCondition issue

### ISSUE THREAD

**Issue by sraju89** [aws-samples/cryoem-on-aws-parallel-cluster#3]
Hello,
I have made by public subnet auto assign Ipv4 addresses but am still running into an issue with the stack. 
The HeadNode is completed but running into an issue with "HeadNodeWaitCondition" with following error: WaitCondition timed out. Received 0 conditions when expecting 1

This is the report from pcluster describe-cluster

"failures": [
    {
      "failureCode": "HeadNodeBootstrapFailure",
      "failureReason": "Cluster creation timed out."
    }
  ]
}

Help would be appreciated

**Comment by bifxcore** [comment]
Anybody there? I'm getting the same error.

The /tmp/postinstall.log shows multiples errors during the post-install process:

Huge stacktrace following the first error:
> dracut module 'multipath' depends on 'dm', which can't be installed

Then second error:
> Job for systemd-modules-load.service failed because the control process exited with error code. See "systemctl status systemd-modules-load.service" and "journalctl -xe" for details.
> 

systemctl status systemd-modules-load.service

returns:

> systemd[1]: systemd-modules-load.service: main process exited, code=exited, status=1/FAILURE
> systemd[1]: Failed to start Load Kernel Modules.
> systemd[1]: Unit systemd-modules-load.service entered failed state.
> systemd[1]: systemd-modules-load.service failed.

**Comment by bifxcore** [comment]
It looks like the error is due to the head node not having any GPUs.

```
# on head node, retrieve PID:
systemctl status systemd-modules-load
# investigate error:
journalctl _PID=<pid>
```
returns 
Failed to insert 'nvidia': No such device

**Comment by rpomaris** [comment]
Addressed in PR #7 by (1) updating the post-install script and (2) increasing the HeadNodeWaitCondition time to 5400 seconds
